company/okrs/2019_q4.md

```diff
@@ -15,8 +15,8 @@
       1. Distribution: Improve site admin experience. Provide additional monitoring and debugging tools to site admins ([#4260](https://github.com/sourcegraph/sourcegraph/issues/4260), [#5547](https://github.com/sourcegraph/sourcegraph/issues/5547), [#5706](https://github.com/sourcegraph/sourcegraph/issues/5706), [#5531](https://github.com/sourcegraph/sourcegraph/issues/5531), [#5369](https://github.com/sourcegraph/sourcegraph/issues/5369), [#5769](https://github.com/sourcegraph/sourcegraph/pull/5769))
       1. Code intel: Deliver LSIF-based code intelligence. Support LSIF-based code intel in targeted languages (Go, JavaScript/TypeScript, Python, Java, C#, and C/C++), LSIF-powered code intel on Sourcegraph.com for at least 1 public project per language.
       1. Automation: Release automation campaign tracking and creation. Implement [RFC 42](https://docs.google.com/document/d/1j85PoL6NOzLX_PHFzBQogZcnttYK0BXj9XnrxF3DYmA/edit).
-      1. Web: Make the power of search accessible to more users. Eliminate errors caused by malformed search queries (e.g. literal search, autocomplete), increase usage of filters (e.g. filter UI).
-      1. Web: Data driven decisions for search improvements. Measure number/type of filters used for each search on customer instances, measure the number of searches that result in an error.
+      1. Web: Make the power of search accessible to more users. Eliminate errors caused by malformed search queries (e.g. literal search, autocomplete), increase usage of filters (e.g. filter UI). => Literal search and autocomplete shipped. Improved filter UI (omni mode, filter highlighting in interactive mode) are expected to ship in Q4.
```

Comment:
KR results are documented to be separated with a comma: https://about.sourcegraph.com/company/okrs#format

```suggestion
      1. Web: Make the power of search accessible to more users. Eliminate errors caused by malformed search queries (e.g. literal search, autocomplete), increase usage of filters (e.g. filter UI). => Literal search and autocomplete shipped, improved filter UI (omni mode, filter highlighting in interactive mode) are expected to ship in Q4.
```

The result does look awkward because it isn't grammatically correct English. I know that @tsenart also had a complaint with this formatting in the context of writing results for core services OKRs. I think we should consider using full stops between KRs and KR results. Proposal: https://github.com/sourcegraph/about/pull/420